### DISCUSSION TITLE
CDK Runway module support

### DISCUSSION THREAD

**Discussion by bali007861** [onicagroup/runway#2022] [General]
We got our first CDK module deployed using Runway, we are heavy cloudformation today and noticed this message on 2.6.11

CDK Runway module support has been deprecated and may be removed in the next major release

What are the plans for Runway and CDK as this would help make our decision on whether we should this for CDK deployments, we would like to stick with runway as we use it everything AWS today.

**Comment by alexander-kogan** [comment]
The long-term support strategy for CDK is currently under evaluation. We will provide a formal announcement regarding future support plans once a decision has been reached.
